CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 ## 0.91.0
```

Comment:
should be 0.90.1. But if we are going to merge #2741 it is fine. It will be fixed by that .